src/Polymesh.ts

```diff
@@ -459,6 +460,40 @@ export class Polymesh {
     });
   }
 
+  /**
+   * Retrieve the event id of the token creation transaction
+   *
+   * @param args.ticker - Security Token ticker
+   */
+  public async getCreationTokenEventId(args: { ticker: string }): Promise<number | null> {
```

Comment:
This shouldn't be exposed in the client root. My bad for not specifying the interface in the task.

This should be part of a `createdAt` function in the SecurityToken entity root. The function should return an object that conforms to

```
// types/index.ts
interface EventIdentifier {
  blockNumber: number;
  eventIndex: number;
}
```

(eventIndex is event_idx in the query results)